### PR TITLE
Router further audit improvements

### PR DESCRIPTION
**Pull request by psorinionut**

### CODE REVIEW

**Review by CostinCarabas — DISMISSED** [review]

File: dex/router/src/config.rs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        );
+    }
+
+    #[view(getPairCreationEnabled)]
+    #[storage_mapper("pair_creation_enabled")]
+    fn pair_creation_enabled(&self) -> SingleValueMapper<bool>;
+
+    #[view(getState)]
+    #[storage_mapper("state")]
+    fn state(&self) -> SingleValueMapper<bool>;
```
Comment (CostinCarabas) [inline dex/router/src/config.rs:25]: A better naming would make using this more easily.
For example, if `state = false` what does this mean?

File: dex/router/src/config.rs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use crate::{enable_swap_by_user::EnableSwapByUserConfig, factory::PairTokens};
+
+#[multiversx_sc::module]
+pub trait ConfigModule {
+    fn is_active(&self) -> bool {
+        self.state().get()
```
Comment (CostinCarabas) [inline dex/router/src/config.rs:9]: Couldn't you have use the pausable module?

File: dex/router/src/config.rs (hunk 1 above)
Comment (psorinionut) [reply dex/router/src/config.rs:25]: Indeed, I agree that the naming isn't the most fortunate, but this is how it was before. I only moved the code in a different module, to be able to call some functions in the multi pair swap module.

File: dex/router/src/config.rs (hunk 2 above)
Comment (psorinionut) [reply dex/router/src/config.rs:9]: The same as before, this was the code from the beginning and I would not change it because of backwards compatibility.

**Review by alyn509 — APPROVED** [review]

**Review by alyn509 — APPROVED** [review]

**Pull request merged**